src/components/LanguagePicker/LanguagePicker.tsx

```diff
@@ -31,46 +26,68 @@ interface Props {
 }
 
 function LanguagePicker({ navigation }: Props) {
+  const [locale, setLocale] = React.useState('is');
+  const selectedLanguage = languages.find(lang => lang.code === locale);
+
   const changeLocale = (locale: string) => () => {
     changeLanguage(locale);
     navigation.navigate('Welcome');
   };
 
   return (
-    <AllLanguages>
-      {languages.map(({ code, title, description, button, flag }, index) => (
-        <Language index={index} key={code}>
-          <LanguageContent>
-            <Text type="heading" level={2} marginBottom={0.25}>
-              {title}
-            </Text>
-            <Text
-              style={{
-                fontSize: verticalScale(16),
-                lineHeight: verticalScale(16 * 1.4),
-                marginBottom: verticalScale((16 * 1.4) / 2),
-              }}
-            >
-              {description}
-            </Text>
-            <ButtonGroup>
-              <CtaButton
-                align="left"
-                justify="start"
-                onPress={changeLocale(code)}
-                image={flag}
-                imageDimensions={{
-                  width: scale(28),
-                  height: scale(19),
-                }}
-              >
-                {button}
-              </CtaButton>
-            </ButtonGroup>
-          </LanguageContent>
-        </Language>
-      ))}
-    </AllLanguages>
+    <LanguageScreen>
```

Comment:
Is this View needed? Or could we use a fragment instead `<></>`.

Reply:
We can definitely use a fragment. I decided to keep some elements with no styles but with a descriptive name mostly for better semantics. However, we can totally skip them. Just wanted to give you my reasoning, but up to you. Just tell me which option you prefer :)